tracee/tracee.bpf.c

```diff
@@ -233,8 +235,8 @@ BPF_HASH(chosen_events_map, u32, u32);              // Events chosen by the user
 BPF_HASH(pids_map, u32, u32);                       // Save container pid namespaces
 BPF_HASH(args_map, u64, args_t);                    // Persist args info between function entry and return
 BPF_HASH(ret_map, u64, u64);                        // Persist return value to be used in tail calls
-BPF_HASH(uid_equal_filter, u32, u32);               // Used to filter events by UID, for specific UIDs either by == or !=
-BPF_HASH(uid_compare_filter, u32, u32);             // Used to filter events by UID, for ranges of UIDs by < or >
+BPF_HASH(uid_equal_filter, u32, int64_t);             // Used to filter events by UID, for specific UIDs either by == or !=
+BPF_HASH(uid_compare_filter, u32, int64_t);           // Used to filter events by UID, for ranges of UIDs by < or >
```

Comment:
I think you can use s64 to keep types definitions in the same convention

Reply:
Ah, ok. To be honest i'm not that familiar with C type differences like this example, something to learn more about.